Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -658,6 +682,38 @@ WarpX::ComputeEightWaysExtensions() {
 #endif
 }
 
+void
+WarpX::ApplyBCKCorrection(const int idim) {
+#if defined(AMREX_USE_EB) and !defined(WARPX_DIM_RZ)
```

Comment:
```suggestion
#if defined(AMREX_USE_EB)
```
How about allowing this for RZ for now? (even though RZ does not support EB for the moment)
Otherwise, I am afraid we may forget about this part of the code, once we add RZ support...